packages/web/src/layouts/AdrBrowserLayout/AdrBrowserLayout.tsx

```diff
@@ -250,18 +254,37 @@ export function AdrBrowserLayout({
   const router = useRouter();
 
   const [mobileDrawerOpen, setMobileDrawerOpen] = React.useState(false);
+  const [selectedPackage, setSelectedPackage] = React.useState("all");
+  const [packages, setPackages] = React.useState(["all"]);
+  const [adrsToDisplay, setAdrsToDisplay] =  React.useState(adrs);
 
   const handleMobileDrawerToggle = () => {
     setMobileDrawerOpen(!mobileDrawerOpen);
   };
+  
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Material UI v5 appears to have a proper SelectChangeEvent type for this, but Material v4 doesn't appear to, so had to turn off a couple of ESLint violations